src/XrdSciTokens/XrdSciTokensAccess.cc

```diff
@@ -339,14 +412,20 @@ class XrdAccSciTokens : public XrdAccAuthorize, public XrdSciTokensHelper
                     access_rules.reset(new XrdAccRules(now + cache_expiry, username, token_subject, issuer, map_rules, groups));
                     access_rules->parse(rules);
                 } else {
+                    m_log.Log(LogMask::Warning, "Access", "Failed to generate ACLs for token");
                     return OnMissing(Entity, path, oper, env);
                 }
+                if (m_log.getMsgMask() & LogMask::Debug) {
+                    m_log.Log(LogMask::Debug, "Access", "New valid token", access_rules->str().c_str());
+                }
             } catch (std::exception &exc) {
-                m_log.Emsg("Access", "Error generating ACLs for authorization", exc.what());
+                m_log.Log(LogMask::Info, "Access", "Error generating ACLs for authorization", exc.what());
```

Comment:
Why is this message being relegated to info? It seems hat one would want to know this has happened, yes?

Reply:
Sounds reasonable.  Fixed in the latest.